src/styles/_Header.scss

```diff
@@ -1,9 +1,9 @@
 .header {
-  // background-image: url(header-background.png);
-  // background-size: cover;
-  // background-repeat: no-repeat;
-  border-bottom: 2px solid tomato;
```

Comment:
No more tomato!? I don't know about this change.

Reply:
🍅

Reply:
It was clashing too much with my hover state that was also conveniently tomato 🤓